### ISSUE TITLE
Vid2Vid different resolution

### ISSUE THREAD

**Issue by CesarCruzUtec** [volotat/SD-CN-Animation#33]
I was trying to do vid2vid on a video with different resolution than 1024x576 , but it only generates the 1st frame and then it stops. 
![image](https://user-images.githubusercontent.com/67798431/236770537-69c3bdf6-aedc-472d-a101-18be5982845f.png)

**Comment by volotat** [comment]
Could you please show the error you are having?

**Comment by CesarCruzUtec** [comment]
It wasn't having errors. It just didn't generate any more frames after the 1st one. But now, I tried to replicate the error and it crashed, neither the interrupt or generate button worked anymore so I restarted it. But it still doesn't generate more from the 1st frame. The only thing changed were the prompt and the size
![image](https://user-images.githubusercontent.com/67798431/236773978-9d74eadf-b935-44ff-84ac-86ce975188a4.png)
Here is the ControlNet settings.
![image](https://user-images.githubusercontent.com/67798431/236774230-438c0141-6c3b-4438-a042-069fecd6fa49.png)
Just to clarify, it worked if using the default resolution, I even used that resolution with this "tall" video and it works, but obviously the video will be strechted to match the size

**Comment by ZaneKearney** [comment]
This is the error I'm getting

`>sdcn_anim_tmp.prepared_next_flows[cn] = next_flow.copy()
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ValueError: could not broadcast input array from shape (848,480,2) into shape (846,480,2)`

**Comment by ZaneKearney** [comment]
This is the error I'm getting when the resolutions don't match:

```
sdcn_anim_tmp.prepared_next_flows[cn] = next_flow.copy()
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ValueError: could not broadcast input array from shape (848,480,2) into shape (846,480,2)
```

**Comment by volotat** [comment]
@ZaneKearney Seems like RAFT does not support arbitrary resolution of images.. I should do some pre and post processing when computing the flow to fix this. Will fix it at the next version. For now I would recommend using multiples of 16 for width and height.

**Comment by volotat** [comment]
Should be fixed in v0.8. Please open new issue if the error still occurs.
